src/EnergyPlus/DaylightingManager.cc

```diff
@@ -2142,8 +2142,8 @@ namespace EnergyPlus::DaylightingManager {
                 }
                 if (ExtWinType == DataDaylighting::iExtWinType::AdjZoneExtWin) {
                     // Does ray pass through an interior window in zone (ZoneNum) containing the ref point?
-                    for (IntWin = Zone(ZoneNum).WindowSurfaceFirst; IntWin <= Zone(ZoneNum).WindowSurfaceLast; ++IntWin) {
-                        if (Surface(IntWin).ExtBoundCond >= 1) {
+                    for (IntWin = state.dataHeatBal->Zone(ZoneNum).WindowSurfaceFirst; IntWin <= state.dataHeatBal->Zone(ZoneNum).WindowSurfaceLast; ++IntWin) {
+                        if (Surface(IntWin).ExtBoundCond >= 1) { // in develop this was Surface(IntWin).Class == SurfaceClass::Window && Surface(IntWin).ExtBoundCond >= 1
```

Comment:
@Myoldmopar Right, the `if==Window` is redundant now, because the loop is over just window surfaces.